### DISCUSSION TITLE
quick/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#773] [Announcements]
# quick/

PageNow前端源码已完全开源，可直接前往码云进行源码拉取。请记得给个免费的小星星，支持一下作者！！！ 前端源码gitee地址：https://gitee.com/jman325_admin/pagenow_open 环境准备 源码拉取后，必须确保本地安装了Nodejs，目前PageNow开发阶段使用的版本为v12.18.3，如果您本地已安装有Nod...

http://localhost:8080/quick/

**Comment by JManToGithub** [comment]
不错
